### PR TITLE
wassup biyoch

### PR DESCRIPTION
**Pull request by upoudel888**
Prajesh dai herdinu yarr

### CODE REVIEW

File: wassup.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+wassup biyoch
```
Comment (proghosta) [inline wassup.txt:1]: bhayena vai pherii garera aija pherii gaarrr

Comment (proghosta) [reply wassup.txt:1]: noice

Comment (proghosta) [reply wassup.txt:1]: noice

Comment (upoudel888) [reply wassup.txt:1]: vagg beyy chicken.. I quit

**Review by proghosta — CHANGES_REQUESTED** [review]
ali kati ramro basha lekh

**Pull request merged**